examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/TagTweet.scala

```diff
@@ -19,7 +19,7 @@ object TagTweet {
   @throws[UnknownPlaceException]
   @throws[TwitterException]
   def tagOneTweet(ln: String, requireGeoField: Boolean): String = {
-    val adm = Tweet.toADM(TwitterObjectFactory.createStatus(ln), usGeoGnosis, requireGeoField)
+    val adm = Tweet.toADM(ln, usGeoGnosis, requireGeoField)
```

Comment:
Why this change is necessary?

Reply:
The "TwitterObjectFactory.createStatus" using Twitter JSON parser, we replaced it, so just need pass the string line to "toADM".